include/picongpu/particles/synchrotronPhotons/PhotonCreator.hpp

```diff
@@ -345,8 +346,8 @@ struct PhotonCreator
      * \tparam Electron type of electron which creates the photon
      * \tparam Photon type of photon that is created
      */
-    template<typename Electron, typename Photon>
-    DINLINE void operator()(Electron& electron, Photon& photon) const
+    template<typename Electron, typename Photon, typename T_Acc>
```

Comment:
OPTIONAL:
according to the [coding guidelines](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#17-template-class--struct-and-type-definitions) each template argument should be on a separate line